tests/test_params_pickling/test.py

```
@@ -0,0 +1,11 @@
+import numpy as np
+import polars as pl
+import pandas as pd
+
+assert type(snakemake.params.testnp) == np.ndarray
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)